sematic/cli/cli.py

```diff
@@ -26,7 +26,7 @@ def cli(verbose: int):
     elif verbose > 1:
         logging.basicConfig(level=logging.DEBUG)
 
-    switch_env("local")
+    # switch_env("local")
```

Comment:
This is a blocker that will be have to be solved in order to submit pipelines natively to non-local Servers.

Reply:
I resurrect the proposal of supporting several profiles in the settings, not only "default".

This would also support several cloud envs, not only local vs one specific cloud env. This is usable by users as well, as they would realistically have dev, staging, and prod envs as well.

So instead of `"use-server"`, you would specify the profile, identifying which server.

Reply:
What about something like
```
# Use default profile
$ sematic run main.py

# Use foo profile
$ sematic run main.py --profile foo
```